cmd/root.go

```diff
@@ -268,7 +268,7 @@ func getExtractPathForName(name string) (string, error) {
 
 func includeLayers() bool {
 	for _, t := range types {
-		if t == "layer" {
+		if strings.Contains(t, "layer") {
```

Comment:
I think we should keep this the way it was, this won't match up with the logic to retrieve the analyzers from the map in `checkIfValidAnalyzer()`. I'm guessing you did this so we could allow `--types layers`? We could add that as a valid string as well, but this would also match `--types asdfasdfbnlayer` which we probably shouldn't do.

Reply:
Well I included it to distinguish the Analyzers that require layer extraction from the ones that don't require it. I noticed the layers were only extracted for the `layer` analyzer, thus I though that enabling the layer extraction for any analyzer including `layer` in its name could be a generic way to distinguish them from the rest in a simple string test. Obviously that assumes that any analyzer requiring extracted layers is named using the `.*layer.*` pattern. At the time of writing it I was having in mind `layer`, `aptlayer` and `rpmlayer` analyzers. After all `checkIfValidAnalyzer` runs first, thus at the time of running `includeLayers` we can realy on having an already valid list of types to check if there is one requiring layers extraction.

It would be also fine for me to use a static list of types that do require layer extraction.